### ISSUE TITLE
[Bug]: Fluent UI v9 needs to provide a way to style using :focus-visible

### ISSUE THREAD

**Issue by jurokapsiar** [microsoft/fluentui#22622]
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
Consumers are not able to style focus indicator with prior Electron vresions
```

### Are you reporting Accessibility issue?

yes

### Reproduction

Consumers are not able to style focus indicator with prior Electron vresions

## Actual Behavior
:focus-visible or similar equivalent not available for styling of focus indicator

## Expected Behavior
there should be a way to style focus indicator

### Requested priority

Blocking

### Products/sites affected

teams

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by bsunderhus** [comment]
Conclusions for this issue are: v9 uses [tabster](https://github.com/microsoft/tabster) for focus indication instead of depending on `:focus-visible`, for third parties to use the same mechanism we must provide better documentation on `react-tabster` for `createFocusOutlineStyle` and `createCustomFocusIndicatorStyle`.
